diff --git a/ios/LocalizationModule.swift b/ios/LocalizationModule.swift
index 73b2b9d5a194141a3b9dd2c98c277707274151e2..c7646880da0877d286d80ec2262a085582bc26b0 100644
--- a/ios/LocalizationModule.swift
+++ b/ios/LocalizationModule.swift
@@ -123,6 +123,8 @@ public class LocalizationModule: Module {
       return "roc"
     case .iso8601:
       return "iso8601"
+    @unknown default:
+      return "gregorian"
     }
   }
 
